### ISSUE TITLE
[Raid] ICC Blood Prince Council 

### ISSUE THREAD

**Issue by Plege** [WoWManiaUK/Redemption#1169]
The following issue can occur after a wipe on Blood Prince Council.

I occasionally happens that one of the three after the wipe (reset) is untargetable which makes him not tankable and he just runs through the raid and kills of one after the other. It's not always the same that can't be targeted and if you try to reset the encounter (rumors have it resetting the encounter several times can solve the issue but i can't confirm that) with a hunter pet or the like it can be that another prince becomes untargetable. it is always just one but it breaks the encounter.

Also after reset they sometimes don't go back to the initial position. This is not a huge problem but a detail.

In short, after a wipe one of the princes is sometimes untargetable. Sometimes they don't go back to initial position.
 
Prince Valanar   37970
Prince Keleseth 37972
Prince Taldaram 37973

Regards
Pleg
